db/rdbms/src/main/resources/mapper/JobMapper.xml

```diff
@@ -12,18 +12,200 @@
+        <foreach collection="filter.elementInstanceKeyOperations" item="operation">
+          AND ELEMENT_INSTANCE_KEY
+          <include refid="io.camunda.db.rdbms.sql.Commons.operationCondition"/>
+        </foreach>
+      </if>
+      <if test="filter.tenantIdOperations != null and !filter.tenantIdOperations.isEmpty()">
+        <foreach collection="filter.tenantIdOperations" item="value" open="(" separator="," close=")">
+          AND TENANT_ID
+          <include refid="io.camunda.db.rdbms.sql.Commons.operationCondition"/>
+        </foreach>
+      </if>
+    </trim>
+  </sql>
+
+  <resultMap id="searchResultMap" type="io.camunda.db.rdbms.write.domain.JobDbModel">
+    <constructor>
+    <idArg column="JOB_KEY" javaType="java.lang.Long" />
+    </constructor>
+    <result column="PARTITION_ID" property="partitionId" javaType="java.lang.Integer" />
+    <result column="PROCESS_INSTANCE_KEY" property="processInstanceKey" javaType="java.lang.Long" />
+    <result column="ELEMENT_INSTANCE_KEY" property="elementInstanceKey" javaType="java.lang.Long" />
+    <result column="PROCESS_DEFINITION_ID" property="processDefinitionId" javaType="java.lang.String" />
+    <result column="PROCESS_DEFINITION_KEY" property="processDefinitionKey" javaType="java.lang.Long" />
+    <result column="TENANT_ID" property="tenantId" javaType="java.lang.String" />
+    <result column="TYPE" property="type" javaType="java.lang.String" />
+    <result column="WORKER" property="worker" javaType="java.lang.String" />
+    <result column="STATE" property="state" javaType="io.camunda.db.rdbms.write.domain.JobDbModel$JobState" />
+    <result column="RETRIES" property="retries" javaType="java.lang.Integer" />
+    <result column="ERROR_MESSAGE" property="errorMessage" javaType="java.lang.String" />
+    <result column="ERROR_CODE" property="errorCode" javaType="java.lang.String" />
+    <result column="END_TIME" property="endTime" javaType="java.time.OffsetDateTime" />
+    <result column="CUSTOM_HEADERS" property="serializedCustomHeaders" javaType="java.lang.String" />
+    <result column="KIND" property="kind" javaType="io.camunda.db.rdbms.write.domain.JobDbModel$JobKind" />
+    <result column="ELEMENT_ID" property="elementId" javaType="java.lang.String" />
+    <result column="IS_DENIED" property="isDenied" javaType="java.lang.Boolean" />
+    <result column="DENIED_REASON" property="deniedReason" javaType="java.lang.String" />
+    <result column="LISTENER_EVENT_TYPE" property="listenerEventType" javaType="io.camunda.db.rdbms.write.domain.JobDbModel$ListenerEventType" />
+    <result column="DEADLINE" property="deadline" javaType="java.time.OffsetDateTime" />
+    <result column="HAS_FAILED_WITH_RETRIES_LEFT" property="hasFailedWithRetriesLeft" javaType="java.lang.Boolean" />
+    <result column="HISTORY_CLEANUP_DATE" property="historyCleanupDate" javaType="java.time.OffsetDateTime" jdbcType="TIMESTAMP"/>
```

Comment:
💭 The other date attributes have the same DB type. Should they also have the same definition as this attribute here in the `searchResultMap`, i.e., including `jdbcType="TIMESTAMP"`? Same goes for the insert and update statements.

Reply:
Can we clarify this with the RDBMS team? Would be great to have a consistent code base in that regard so we don't copy-past different things.